src/libs/OptionsListUtils.js

```diff
@@ -62,7 +62,7 @@ const iouReports = {};
 Onyx.connect({
     key: ONYXKEYS.COLLECTION.REPORT,
     callback: (report, key) => {
-        if (!report || !key || !report.ownerEmail) {
+        if (!report || !key || !report.ownerAccountID) {
```

Comment:
I thinkkkk we're setting `ownerAccountID` to `0` here 'n there (where the email was `__FAKE__`) right? I'll make sure this doesn't always `return` if `ownerAccountID` is `0`

Reply:
Ahhh good catch.